lib/activejob/lockable/lockable.rb

```diff
@@ -70,6 +64,11 @@ def lock_period
         options[:lock].to_i
       end
 
+      def trigger_on_locked_action
+        logger.info "Job is locked, expires in #{locked_ttl} second(s)"
+        send(on_locked_action) if on_locked_action && respond_to?(on_locked_action)
```

Comment:
Could this be `public_send`, `send` makes me uneasy :(

Reply:
Sure thing! Done on 60cd0b6